### PR TITLE
Feature/prepare ios version

### LINKED ISSUE #52: Fix googleplus login on Ios
closed #49

### LINKED ISSUE #53: GoogleAnalytics file not found during build Ios
'GoogleAnalytics/GAI.h' file not found
cause a build failure using Xcode for Ios

### LINKED ISSUE #54: Fix Voice Recorder - IOS

### LINKED ISSUE #55: Set App icons to iOS platform - iOS

### PR DESCRIPTION
**Pull request by tomivm**
Should be tested for android Builds

The steps to work are:
A) Install cocoapods using `sudo gem install cocoapods` 
Close #53 
B) On Xcode open `AAC Cboard.xcworkspace`
    1) Select a group for the iOS sign-in certificate.

    2) Add the environment variables for the 'Google Plus Plugin' to XCODE:
        Open your Xcode project and select the project from the project navigator.
        Select the "Build Settings" tab in the main area of the window.
        Search for "User-Defined" in the search bar.
        Click the "+" button to add a new user-defined setting.
        Enter the name of the environment variable you want to set, such as MY_ENV_VAR.
        Enter the value of the environment variable you want to set, such as my-value.
        Save the changes.

        Using this plugin, it is not possible to use the device simulator (at least for now).
Close #52 

   3) Some targets within pods will require enabling 'bitcode', including one in AAC Cboard => libCordova.a.
   
   4) Set a swift-language-version >= 4.2 . This should be settled on AAC Cboard Target under swift compiler- language Close #54 
   5) Update to recommended settings and not allow changes on 'always embed swift standards libraries' to not break the voice record feature
image

close #55

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,19 +22,20 @@
...
     "cordova-electron": "^3.0.0",
+    "cordova-ios": "^6.2.0",
     "cordova-plugin-android-permissions": "^1.1.3",
     "cordova-plugin-androidx": "^3.0.0",
     "cordova-plugin-androidx-adapter": "^1.1.3",
     "cordova-plugin-file": "^6.0.2",
     "cordova-plugin-file-opener2": "^2.2.1",
     "cordova-plugin-fullscreen": "^1.3.0",
-    "cordova-plugin-google-analytics": "git+https://github.com/danwilson/google-analytics-plugin.git",
+    "cordova-plugin-google-analytics": "github:danwilson/google-analytics-plugin",
```
Comment (RodriSanchez1) [inline package.json:32]: Rollback these changes, if not the circleci build will fail

File: package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "cordova-browser": "^6.0.0",
     "cordova-plugin-facebook-connect": "3.2.0",
+    "cordova-plugin-googleplus": "8.5.2",
```
Comment (RodriSanchez1) [inline package.json:15]: The same package is on `devDependecies`. Leave just one of them

File: package.json [hunk 3]
```diff
@@ -29,7 +30,7 @@
     "cordova-plugin-fullscreen": "^1.3.0",
     "cordova-plugin-google-analytics": "git+https://github.com/danwilson/google-analytics-plugin.git",
     "cordova-plugin-googleplus": "^8.5.2",
-    "cordova-plugin-market": "git+https://github.com/xmartlabs/cordova-plugin-market.git",
+    "cordova-plugin-iosrtc": "^8.0.3",
```
Comment (RodriSanchez1) [inline package.json:33]: Add `cordova-plugin-market` again! We use that plugin

**Comment by tomivm** [comment]
record sounds for tiles continue with issues. I will put this PR to draft again.

**Review by martinbedouret — APPROVED** [review]

**Pull request merged**